src/Data/Functor/Day.purs

```diff
@@ -94,6 +96,11 @@ assoclDay = runDay \phi f x -> runDay (\psi g h -> day identity (day (\a b c ->
 assocrDay :: forall f g h. (f ⊗ g) ⊗ h ~> f ⊗ (g ⊗ h)
 assocrDay = runDay \phi x h -> runDay (\psi f g -> day (#) f (day (\a b c -> phi (psi c a) b) g h)) x
 
+tupleDay :: forall f g x y . f x -> g y -> Day f g (Tuple x y)
```

Comment:
I feel like this maybe crosses the [Fairbairn threshold](https://mail.haskell.org/pipermail/libraries/2012-February/017548.html). I'm curious what you use it for?

Reply:
This was mainly to facilitate the above example.  Will happily delete!